e2e/suites/list-views/tooltips.test.ts

```diff
@@ -175,46 +175,6 @@ describe('File / folder tooltips', () => {
     });
   });
 
-  // disabled until ACA-518 is done
-  xdescribe('on Shared Files', () => {
```

Comment:
please do not delete those. They need to be excluded properly and someone should look at the jira issue, since that sounds like a bug with the nodeNameTooltip pipe

Reply:
I will modify the ticket to create the unit test for it they have not to be e2e